bin/ChakraCore/ChakraCore.def

```diff
@@ -38,3 +38,9 @@ JsTTDNotifyYield
 JsTTDPrepContextsForTopLevelEventMove
 JsTTDMoveToTopLevelEvent
 JsTTDReplayExecution
+
+JsInitializeModuleRecord
+JsParseModuleSource
+JsModuleEvaluation
+JsSetModuleHostInfo
+JsGetModuleHostInfo
```

Comment:
you should consider adding nativetests for these sets of api. nativetests should be the place for adding negative tests to validate the various inputs for these APIs which ch.exe is not supposed to do.